src/styles/utills/mixins.scss

```diff
@@ -0,0 +1,24 @@
+@mixin arrow ($animDuratin) {
+  background-color: black;
+  border-radius: 20px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform-origin: bottom;
+  transform: translate(-50%, -100%);
+
+  animation-duration: $animDuratin;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
```

Comment:
Do not forget about good formatting of the code and separate the code with an empty line where necessary.